implementations/elixir/ockam/ockam/lib/ockam/telemetry_logger.ex

```diff
@@ -6,21 +6,29 @@ if Code.ensure_loaded?(:telemetry) do
     require Logger
 
     def handle_event(event, measurements, metadata, _config) do
+      ockam_workers = Ockam.Node.list_workers()
```

Comment:
Let's not add this to the log, it's not that helpful.

Reply:
Ok. Removed from log message.